lib/src/dto/remote_config.dart

```diff
@@ -14,9 +15,13 @@ class QRemoteConfig {
   @JsonKey(name: 'experiment')
   final QExperiment? experiment;
 
+  @JsonKey(name: 'source')
+  final QRemoteConfigurationSource source;
```

Comment:
Doublecheck - neither this field nor it's nested fields are nullable, right?

Reply:
Right